static/docs/user-guide/contributing-documentation.md

```diff
@@ -0,0 +1,113 @@
+# Contributing Documentation
+
+We welcome contributions to [DVC.org](https://github.com/iterative/dvc.org) by
+the community.
+
+## How to report a problem
+
+Please search [issue tracker](https://github.com/iterative/dvc.org/issues)
+before creating a new issue. This issue can be about a typographical error, an
+updation of the documentation or simply an improvement in generation of
+documentation.
+
+If you feel that you can fix or implement it, please read a few paragraphs below
+to learn how to submit your changes.
+
+## Structure of project
+
+* [Content](https://github.com/iterative/dvc.org/tree/master/static/docs) -
+  static Markdown files that are rendered dynamically in the browser. Merging
+  changes to these files into the master branch is enough to update the docs and
+  redeploy the website.
+
+* [Media](https://github.com/iterative/dvc.org/tree/master/static/img) - add new
+  images, gifs, diagrams here. Reference them from the Markdown files like this:
+  `![](/static/img/reproducibility.png)`.
+
+* [Sections](https://github.com/iterative/dvc.org/tree/master/src/Documentation/sidebar.json) - 
```

Comment:
This paragraph isn't formatted properly since it wasn't being rendered correctly. Therefore, I formatted it manually.

#### After formatting
![Screenshot from 2019-05-11 12-02-03](https://user-images.githubusercontent.com/35191225/57566212-8d7e1200-73e7-11e9-81c2-bd4c570d0499.png)